apps/laboratory/src/components/AppKitInfo.tsx

```diff
@@ -30,7 +30,7 @@ export function AppKitInfo({ caipAddress, address, chainId, clientId }: AppKitIn
             <Heading size="xs" textTransform="uppercase" pb="2">
               CAIP Address
             </Heading>
-            <Text data-testid="appkit-caip-address">{caipAddress}</Text>
+            <Text data-testid="w3m-caip-address">{caipAddress}</Text>
```

Comment:
should this stay `appkit-caip-address`?

Reply:
Since this is laboratory it's not mandatory - it's just annoying that others have `w3m-` and only this one has `appkit-`. I've just aligned them with the other laboratory elements. Something outside of the AppKit